package.json

```diff
@@ -18,7 +18,6 @@
     "clean": "yarn lerna run clean",
     "lint": "yarn lerna run lint --stream",
     "test": "yarn lerna run test --stream --concurrency 1",
-    "browserify": "yarn lerna run browserify",
```

Comment:
`browserify` is specific to `near-api-js` and _may_ not be used in future packages, I feel it is better to encapsulate the entire build under `yarn build` and have all packages implement their custom build steps under that.

Reply:
Just double checking - this isn't used by any existing scripts right? I may have included it here for that purpose

Reply:
This is used in `.travis.yml` but looks like Travis isn't connected to the project anymore, we can probably remove this file.

Reply:
Do we need to check this with previous maintainers? If not I say let's remove it 👍  cc @volovyks